noxfile.py

```diff
@@ -93,6 +102,27 @@ def lint(session):
     )
     session.run("flake8", "setup.py", "docs", "dummyserver", "src", "test")
 
+    # TODO: When all files are typed we can change this to .run("mypy", "--strict", "src/urllib3/")
+    session.log("mypy --strict src/urllib3/")
+    for typed_file in TYPED_FILES:
+        if not os.path.isfile(typed_file):
+            session.error(f"The file {typed_file!r} couldn't be found")
+        popen = subprocess.Popen(
+            f"mypy --strict --follow-imports=skip {typed_file}",
```

Comment:
The default mypy behavior is to follow imports, this flag disables it as per https://mypy.readthedocs.io/en/latest/running_mypy.html#following-imports.